frontend/src/app/components/ExperimentCardList.js

```diff
@@ -1,46 +1,44 @@
 import React from 'react';
-import moment from 'moment';
+
+import { isExperimentEnabled } from '../reducers/addon';
+
 import ExperimentRowCard from './ExperimentRowCard';
 import Loading from './Loading';
 
-export default function ExperimentCardList({
-  navigateTo, isExperimentEnabled, hasAddon, isDev, experiments, except, eventCategory
-}) {
-  const experimentsSorted = [].concat(experiments);
-
-  experimentsSorted.sort((a, b) => {
-    if (a.order > b.order) { return 1; }
-    if (a.order < b.order) { return -1; }
-    return 0;
-  });
+export default class ExperimentCardList extends React.Component {
+  getExperiments() {
```

Comment:
Since you're moving things into separate selectors, would this `except` filtering logic make sense to live there too? Or maybe no, because I think this card list is the only place we use it - so maybe just display logic?